### PR TITLE
Add `GenerativeModel.count_tokens` demonstration

### PR DESCRIPTION
**Pull request by mayureshagashe2105**
Adds "Counting number of Tokens" section to python_quickstart to demonstrate how to use the `GenerativeModel.count_tokens` method. As discussed earlier , this section gives a brief about `Tokens`

## Type of change
Documentation

## Checklist

- [x] I have performed a self-review of my code.
- [x] I have added detailed comments to my code where applicable.
- [x] I have verified that my change does not break existing code.
- [x] My PR is based on the latest changes of the main branch (if unsure, please run `git pull --rebase upstream main`).
- [x] I am familiar with the [Google Style Guide](https://google.github.io/styleguide/) for the language I have coded in.
- [x] I have read through the [Contributing Guide](https://github.com/google/generative-ai-docs/blob/main/CONTRIBUTING.md) and signed the [Contributor License Agreement](https://cla.developers.google.com/about).

### CODE REVIEW

**Comment by markmcd** [comment]
Can you run nbfmt on this so the diff renders properly. Should be details in the failing check.

**Comment by mayureshagashe2105** [comment]
> Can you run nbfmt on this so the diff renders properly. Should be details in the failing check.

@markmcd, done. Thank you

**Review by MarkDaoust — COMMENTED** [review]
LGTM. 

@pcoet any feedback on the text?

File: site/en/tutorials/python_quickstart.ipynb [hunk 1]
```diff
@@ -1034,6 +1034,66 @@
         "  display(to_markdown(f'**{message.role}**: {message.parts[0].text}'))"
       ]
     },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "AEgVOYu0pAr4"
+      },
+      "source": [
+        "## Counting number of Tokens\n",
```
[the remark this answers is not in the document]
Comment (8bitmp3) [reply site/en/tutorials/python_quickstart.ipynb:1043]: Nit: "tokens" (sentence case)

File: site/en/tutorials/python_quickstart.ipynb [hunk 2]
```diff
@@ -1034,6 +1034,66 @@
...
+        "\n",
+        "All language models have a context window which is often mentioned in terms of **number of tokens**. In order to utilize the full potential of these language models, determining the number of tokens involved in `ChatSessions` or `prompts` is crucial.\n",
+        "\n",
+        "Gemini enables you to count the number of tokens involved in any `glm.Content` object. In the simplest case, you can pass a query string to the `GenerativeModel.count_tokens` method:"
```
Comment (8bitmp3) [inline site/en/tutorials/python_quickstart.ipynb:1047]: The "to utilize the full potential" of LLMs, "determining the number of tokens" may be a bit subjective, without providing a clear explanation why it's "crucial". Maybe it's an open area of research.

How about:

```suggestion
        "Large language models have a context window, and the context length is often measured in terms of the **number of tokens**. With the Gemini API, you can determine the number of tokens per any `glm.Content` object. In the simplest case, you can pass a query string to the `GenerativeModel.count_tokens` method as follows:"
```

Comment (8bitmp3) [reply site/en/tutorials/python_quickstart.ipynb:1047]: @MarkDaoust cc

**Review by 8bitmp3 — APPROVED** [review]
LGTM. Please check the comments.

File: site/en/tutorials/python_quickstart.ipynb (hunk 2 above)
Comment (8bitmp3) [reply site/en/tutorials/python_quickstart.ipynb:1047]: @pcoet cc

File: site/en/tutorials/python_quickstart.ipynb [hunk 3]
```diff
@@ -141,7 +141,7 @@
     },
```
Comment (pcoet) [inline site/en/tutorials/python_quickstart.ipynb:141]: <p>"which is often mentioned in terms of" -&gt; "which can be described in terms of"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/generative-ai-docs/pull/227/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='58'/>

File: site/en/tutorials/python_quickstart.ipynb (hunk 2 above)
Comment (pcoet) [reply site/en/tutorials/python_quickstart.ipynb:1047]: @8bitmp3 makes a good suggestion. Feel free to make this change and ignore mine below.

Comment (mayureshagashe2105) [reply site/en/tutorials/python_quickstart.ipynb]: Done with all of the suggested changes. Thank you

File: site/en/tutorials/python_quickstart.ipynb (hunk 1 above)
[the remark this answers is not in the document]
Comment (mayureshagashe2105) [reply site/en/tutorials/python_quickstart.ipynb]: Done. Thank you.

File: site/en/tutorials/python_quickstart.ipynb (hunk 3 above)
Comment (mayureshagashe2105) [reply site/en/tutorials/python_quickstart.ipynb]: > @8bitmp3 makes a good suggestion. Feel free to make this change and ignore mine below.

Ignoring this change . Thank you.

**Comment by 8bitmp3** [comment]
Thanks you @pcoet @mayureshagashe2105 @MarkDaoust

**Pull request merged**